app/models/user.rb

```diff
@@ -1,6 +1,6 @@
 class User < ApplicationRecord
   validates_presence_of :name, :address, :city, :state, :zip, :role
-  validates_presence_of :password, require: true
+  validates_presence_of :password_digest
```

Comment:
This still works because the user's actual password is never stored when looking at params or when calling their attributes, it's only their password_digest.

Reply:
Also our test wasn't passing because when we tried to edit a user's information with information that didn't include their password it would give us the error of "Must have password"

Reply:
good catch